### PR TITLE
Vilna Forest

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -2481,53 +2481,53 @@ QUEST_LV_0200_20150317_002480	{memo X}Obtain Royal Jellies by defeating the mons
...
-QUEST_LV_0200_20150317_002485	Talk to Maras
+QUEST_LV_0200_20150317_002484	$Collect %s from monsters
+QUEST_LV_0200_20150317_002485	$Talk to Maras
 QUEST_LV_0200_20150317_002486	{memo X}Talk to Maras who is investigating at Three Brooks Wood
-QUEST_LV_0200_20150317_002487	Check the container with sweet fragrance
-QUEST_LV_0200_20150317_002488	Maras wants to check whether the thing that the monsters are going after is honey or not. Obtain the pieces of honey that would lure the monsters.
-QUEST_LV_0200_20150317_002489	You've obtained the sweet honey piece. Listen to Maras about the next mission.
-QUEST_LV_0200_20150317_002490	Obtain %s from beehives
-QUEST_LV_0200_20150317_002491	Listen to how Maras will use the pieces of the sweet honey container
+QUEST_LV_0200_20150317_002487	$Check Sweet-smelling Beehives
```
Comment (TentuZero) [inline QUEST_LV_0200.tsv:2487]: Check the sweet-smelling beehives?

**Comment by Sunaries** [comment]
:+1:

File: ETC.tsv [hunk 2]
```diff
@@ -9498,7 +9498,7 @@ ETC_20150317_009498	Linkis Sealed Tower
 ETC_20150317_009499	Linker Sealed Tower
 ETC_20150317_009500	Spring Flower Trigger
 ETC_20150317_009501	Farm
-ETC_20150317_009502	A sweet smelling beehive
+ETC_20150317_009502 	Sweet-smelling Beehive
```
Comment (terrapin47) [inline ETC.tsv]: You have a space + tab here, remove the space after 009502

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -1026,40 +1026,40 @@ QUEST_LV_0200_20150317_001025	{memo X}If it weren't you, we would have never got
...
-QUEST_LV_0200_20150317_001031	I made a request at Klaipeda, but they are not responding since they have a lack of soldiers.{nl}And it is not easy to hire merceneries in this era.{nl}
-QUEST_LV_0200_20150317_001032	Then the ones that are left are the Revelators.{nl}You looking for the Goddess? we have the Goddess in our village too.
-QUEST_LV_0200_20150317_001033	Look carefully on the unbroken bee boxes.{nl}You could maybe obtain some bee boxes that are in good status.
-QUEST_LV_0200_20150317_001034	Oh, this is quite big.{nl}Well done.
-QUEST_LV_0200_20150317_001035	You go back to Bulbini Farm.{nl}If we could provide a proof that it was due to the honey, the priests would not only think about the altar.{nl}
-QUEST_LV_0200_20150317_001036	There are many altars around here.{nl}You know it already right? They are all created to praise Goddess Austeja.
+QUEST_LV_0200_20150317_001029	$Anyways, I've placed all my faith on the priest, but she's quickly losing that faith.{nl}That's why I want to hire someone who is skillful like you. {nl}I will compensate you well.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:1029]: Anyways, I've placed all of my faith in the priest, but she's quickly making me lose that faith.

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -1026,40 +1026,40 @@ QUEST_LV_0200_20150317_001025	{memo X}If it weren't you, we would have never got
...
+QUEST_LV_0200_20150317_001047	$Maras told you to show me this?{nl}This is the very object that the priest stressed to us not to touch, was it broken like this originally?
+QUEST_LV_0200_20150317_001048	$I should talk to Maras about this.{nl}Something doesn't feel right.. what could this be used for?
 QUEST_LV_0200_20150317_001049	Ris
-QUEST_LV_0200_20150317_001050	The monsters ruined all the fields so I should somehow try to start it all over again.{nl}Ah well, I confronted a honeypin so I just ran away without carrying anything.{nl}My agricultural equipment.. I don't know what to do from now.
-QUEST_LV_0200_20150317_001051	Thank you so much.{nl}The place where I lost it is Shirushie Sunny Fields.{nl}Honeypins are huge and scary so don't try too hard if you can't handle them.
-QUEST_LV_0200_20150317_001052	Ah. That's right. That's it.{nl}Everything will be okay if the villagers could take care of the monsters.
+QUEST_LV_0200_20150317_001050	$The monsters ruined all the fields so I was trying to start from scratch again.{nl}And then of all things a Honeypin showed up, so I ran away without carrying anything.{nl}My farming tools.. I don't know what to do now.
```
Comment (terrapin47) [inline QUEST_LV_0200.tsv:1050]: "so I ran away without carrying anything" -> "so I left everything and ran"

**Comment by vtange** [comment]
Good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**